docs/source/tutorial/bond-extension.rst

```diff
@@ -0,0 +1,130 @@
+.. Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Tutorials : How to use the Bond extension package
+#################################################
+
+How to create a bond instrument
```

Comment:
I think in the beginning we need to explain how an application should be including the bond package. Since we don't want an app itself to depend on the bond _implementation_ package, the app should only reference the bond _interface_ package. The initialization scripts of the app (which are only run once when the app is initialized) would then create the corresponding factories, and the app would create bonds through the factory interfaces. See section "Structure of the code, dependencies" in the getting-started tutorial, where this is explained.

Reply:
I added a short section in the beginning of the document, where I also referred both to the Getting Started section and the Architecture page.